### PR TITLE
Maya: allow not creation of group for Import loaders

### PR DESCRIPTION
**Pull request by kalisp**
## Changelog Description
This PR enhances previous one. All ReferenceLoaders could not wrap imported products into explicit group.
Also `Import` Loaders have same options. Control for this is separate in Settings, eg. Reference might wrap loaded items in group, `Import` might not.

## Testing notes:
1. Remove `_GRP` in `project_settings/maya/load/import_loader`
2. Try to `Import` anything into Maya
3. Try import `abc` representation of `animation`

### CODE REVIEW

File: openpype/hosts/maya/api/plugin.py [hunk 1]
```diff
@@ -523,6 +523,49 @@ def remove_instances(self, instances):
...
+                    subset['data']['families'][0]
+            )
+        }
+        custom_namespace = custom_naming['namespace'].format(
+            **formatting_data
+        )
+        custom_group_name = custom_naming['group_name'].format(
+            **formatting_data
+        )
+        return custom_group_name, custom_namespace, options
```
Comment (BigRoy) [inline openpype/hosts/maya/api/plugin.py:567]: To me personally the removed empty lines here in the code actually make this function much harder to read than the original. It's cosmetics and it is an opinion. Just wanted to remark that reading this version feels much more cluttered to me than the original. So if you have no preference, let's keep the original. If you prefer this, you're a maniac - haha, no, do whatever you want. 😈

**Review by BigRoy — COMMENTED** [review]
Code wise this seems fine. I think it's massively overkill to have this amount of customization for an "import loader" since once something is imported it's unmanaged and it's trivial to remove the namespaces and group it - but I guess maybe there are cases where studios import so much stuff all the time that they need to tweak the defaults.

**Comment by moonyuet** [comment]
I tried to import abc representation from animation family but it seems that it doesn't apply the namespace as expected
image

there is no "reference animation"(the animation family loader made from load_animation.py) for loading abc
image

File: openpype/hosts/maya/api/plugin.py (hunk 1 above)
Comment (kalisp) [reply openpype/hosts/maya/api/plugin.py:567]: Hmm, this is totally unwanted, probably copy&paste issue. Will fix that.

Comment (kalisp) [reply openpype/hosts/maya/api/plugin.py:567]: Btw this is client requested, see `sponsored` tag on this PR which denotes that some customer requested that. Who am I to argue :D.

**Comment by moonyuet** [comment]
i tried to test it in the ayon and the setting doesn't show up any host when I set the maya version to 0.1.3, it's working when I switch to 0.1.1(along with other hosts)
image
image

**Review by moonyuet — APPROVED** [review]
image
I tried the import options, it seems working as expected. Thanks for helping out on this.

**Review by iLLiCiTiT — APPROVED** [review]
Code looks ok, can't test.

**Pull request merged**